.github/workflows/community-releases.yml

```diff
@@ -0,0 +1,160 @@
+name: Docker Community Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version to tag docker images'
+        required: true
+        type: string
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    timeout-minutes: 80
+    environment: Production
+    permissions:
+      contents: read
+      packages: write
+      deployments: write
+    steps:
+      - uses: actions/checkout@v4
+
+  docker_build_api:
+    runs-on: ubuntu-latest
+    needs: prepare
+    timeout-minutes: 80
+    environment: Development
+    permissions:
+      contents: read
+      packages: write
+      deployments: write
+      id-token: write
+    strategy:
+      matrix:
+        name: ['novu/api-ee', 'novu/api']
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: ${{ contains (matrix.name,'-ee') }}
+          token: ${{ secrets.SUBMODULES_TOKEN }}
+      - uses: ./.github/actions/setup-project
+        with:
+          submodules: ${{ contains (matrix.name,'-ee') }}
+
```

Comment:
for the community release, we shouldn't use the matrix and build the enterprise packages (submodules)

Reply:
@LetItRock shouldn't we include it in the workflow inputs for self-hosted EE deployments?

Reply:
@SokratisVidros @LetItRock From working with the enterprise clients when we release a community version would should also release a enterprise version.